addons/a11y/README.md

```diff
@@ -18,20 +18,15 @@ Add this line to your `main.js` file (create this file inside your storybook con
 
 ```js
 module.exports = {
-  addons: ['@storybook/addon-a11y/register'],
+  addons: ['@storybook/addon-a11y'],
```

Comment:
Docs should lead with the preset IMO. 

If you want the old behavior of 'off by default'  instead of using register why not set disable: true in main.js and then enable it selectively?